securedrop/source_app/__init__.py

```diff
@@ -106,6 +106,11 @@ def handle_csrf_error(e):
     for module in [main, info, api]:
         app.register_blueprint(module.make_blueprint(config))  # type: ignore
 
+    # Disables the app if the server is running Trusty past its EOL date.
+    app.config['XENIAL_VER'] = "16.04"
```

Comment:
Instead of adding these as config options, we could still mock them by making them constants in the `disable` module.

Reply:
done